library/src/main/java/com/devbrackets/android/exomedia/core/api/VideoViewApi.java

```diff
@@ -94,6 +94,8 @@
 
     void setScaleType(@NonNull ScaleType scaleType);
 
+    void setDoNotMeasureBasedOnAspectRatio(boolean doNotMeasureBasedOnAspectRatio);
```

Comment:
Hmm, typically methods that take booleans (toggle methods) are worded towards enabling something (e.g. `setStateEnabled(...)`); would you re-word it to follow that paradigm?  

e.g.
```java
void setAspectAwareOnMeasureEnabled(boolean enabled);
// or 
void setMeasureBasedOnAspectRatioEnabled(boolean enabled);
// or something similar
```

I know I'm being a little picky on this so thanks for hanging in there :)